fix command (#398)
status-im/ift-docs: docs/basecamp/get-started/install-logos-basecamp.md

1. Run the resulting binary:

    ```bash
    ./result/logos-basecamp.AppImage
    ```

## Troubleshooting Basecamp

> ### experimental Nix feature 'nix-command' when building appimage

Reviewer: Nit: capitalize the heading and use the product's `AppImage` casing (as elsewhere in this doc) for consistency with the other troubleshooting headings.

```suggestion
### Experimental Nix feature 'nix-command' when building the AppImage
```

> Following error might appear while building appimage:

Reviewer: Nit: small grammar polish.

```suggestion
The following error might appear while building the AppImage:
```

> error: experimental Nix feature 'nix-command' is disabled; add '--extra-experimental-features nix-command' to enable it

> in which case command should be changed to allow flakes syntax:

Reviewer: Nit: grammar — add the article and "enable flakes" reads more naturally than "allow flakes syntax".

```suggestion
in which case the command should be changed to enable flakes:
```
